staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -394,7 +394,7 @@ func RecordRequestAbort(req *http.Request, requestInfo *request.RequestInfo) {
 	}
 
 	scope := CleanScope(requestInfo)
-	reportedVerb := cleanVerb(CanonicalVerb(strings.ToUpper(req.Method), scope), getVerbIfWatch(req), req)
+	reportedVerb := cleanVerb(CanonicalVerb(strings.ToUpper(req.Method), scope), "", req)
```

Comment:
OK - the `getVerbIfWatch` is called from CleanVerb (which is called from cleanVerb), so this change doesn't break anything for metrics.
At the same time it's unifying CleanVerb call, which is great.

Thanks for cleaning this up!